.github/CODEOWNERS

```
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj  @king-alexander
```

Maintain alphabetical order and remove extraneous space:
```suggestion
* @dav3r @felddy @jasonodoom @jsf9k @king-alexander @mcdonnnj
```